autogpt/agent/agent.py

```diff
@@ -284,7 +284,10 @@ def signal_handler(signum, frame):
                 memory_tlength = count_string_tokens(
                     str(self.history.summary_message()), self.config.smart_llm
                 )
-                if result_tlength + memory_tlength + 600 > self.smart_token_limit:
+                if str(command_result).startswith("Unknown command"):
```

Comment:
a search for `Unknown command` returned 0 results in the codebase, so there is no way this will work

Reply:
Thanks, Pwuts. It was here: https://github.com/Significant-Gravitas/Auto-GPT/blob/8bce02736b7fcc3ef14809fa935c9f147ccc2e0a/autogpt/app.py#L131 but it seems the issue was fixed with this PR: https://github.com/Significant-Gravitas/Auto-GPT/commit/4177c37b51d2f2c8f19c929cadd2609262682605#diff-2c87cd30823d1d2b44a5130d7d1a788b97928a00ff4dc84e43fbfb0e2f2c1559R394. All good then. Thanks, Pwuts.